switchapi/es2k/switch_pd_fdb.c

```diff
@@ -17,14 +17,14 @@
 
 #include "switch_pd_fdb.h"
 
-#include "bf_types.h"
-#include "port_mgr/dpdk/bf_dpdk_port_if.h"
+#include "port_mgr/dpdk/dpdk_port_if.h"
```

Comment:
Including dpdk_port_if.h in es2k/switch_pd_fdb.c? Please double check

Reply:
[include/port_mgr/dpdk/bf_dpdk_port_if.h → include/port_mgr/dpdk/dpdk_port_if.h]
they have renamed the file and removed "bf_" prefix from the existing filename.

Reply:
Nupur's comment was that you're including **dpdk**/**dpdk**_port_if.h in the **es2k** directory.
- **dpdk** goes with **dpdk**
- **es2k** goes with **es2k**

and never the twain shall meet.

Unless the SDE team screwed up, in which you should add a comment explaining to the reader that this is not an error. Because it bloody well looks like a mistake.

Reply:
The change here is to adapt to the sde header file name change: `bf_dpdk_port_if.h` to `dpdl_port_if.h`.
We shouldn't have included `bf_dpdp_port_if.h` in the first place. 
We need to review this and remove, maybe in a different PR.

Reply:
you are right.. we are not even using any def from this file IMO. please remove and check once.

Reply:
You are right, the file inclusion is unnecessary. Although objective of this PR is not to do cleanup of unnecessary header files, but since the change is small, I have addressed it.